labs/Cardea.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Cardea Proposal Document
+parent: Labs
+---
+# Lab Name
+_Cardea_
+
+
+# Short Description
+_Cardea is a complete ecosystem for the exchange of privacy-preserving digital credentials, currently open sourced as a project in Linux Foundation Public Health (LFPH)._
+
+# Scope of Lab
+_Cardea builds on Hyperledger Aries, Hyperledger Indy, and Hyperledger Ursa to provide an ecosystem for privacy-preserving digital credentials, focused on the intersection of healthcare data and verifiable credentials. Cardea includes a package of agents to facilitate issuance, holding and verification of credentials pertinant to healthcare data. Cardea has hosted several interop-a-thons to facilitate interoperability testing between the Cardea community, as well as the broader Aries community. _
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/dbluhm
+- https://github.com/Eldersonar
+- https://github.com/JamesKEbert
+- https://github.com/KeelaShatzkin
+- https://github.com/lynnbendixsen
+- https://github.com/Megan-Olsen
+- https://github.com/mikekebert
+- https://github.com/reflectivedevelopment
+- https://github.com/ryjones
+- https://github.com/seajensen
+- https://github.com/SeanBohan
+- https://github.com/stevedavisbiz
+- https://github.com/thelinuxfoundation
+- https://github.com/TheTreek
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is optional, but the sponsor must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair. Read about sponsors' duty in [Section 3, Labs proposal](./index.md#process-to-propose-a-new-lab)._
+- https://github.com/TelegramSam (Co-Chair of Hyperledger Aries working group)
+
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
+- https://github.com/thecardeaproject
```

Comment:
This is the first lab that we have had that have asked to move an entire organization. Are you suggesting that we would move all of the repos in this organization to the hyperledger-labs organization?

Reply:
Yes, we would like to move the entire project, including all of the agents and their respective repos.

Reply:
It looks like there are 10 repositories (I would not recommend moving governance, as we have our own governance document). There are some discrepancies with LFPH CoC and the Hyperledger CoC that we will probably want to resolve once it is moved into place. @ryjones : Do you see any issues with moving over the 10 cardea* repos?

Reply:
@tkuhrt nope, I've been waiting to do it for a few months